dnf-behave-tests/features/steps/common/rpm.py

```diff
@@ -100,12 +100,20 @@ def diff_rpm_lists(list_one, list_two):
         "changed": [],
         "unchanged": [],
         "present": [],
-        "absent": [],
     }
 
     list_one = sorted(list_one)
     list_two = sorted(list_two)
 
+    # PRESENT PACKAGES AND UNCHANGED INSTALLONLY PACKAGES
+    for pkg in list_two:
+        result["present"].append(pkg)
+        if (pkg in list_one) and pkg.is_installonly():
+            result["unchanged"].append(pkg)
+    for pkg in result["unchanged"]:
```

Comment:
Reinstall == unchanged?

Reply:
I'm not sure how to distinguish these two states. When comparing two versions of rpm database, both look the same. We probably should not bother with unchanged/reinstall on rpmdb level and consider them one state. This definitely deserves more work.